src/pages/consequences.module.scss

```diff
@@ -0,0 +1,56 @@
+@import '../assets/shared-styles/colors.module.scss';
+
+.riskBuying {
+    display: inline-block;
+    font-family: cera-light;
+    font-size:180%;
+    width:300px;
+}
+
+.offenderRegistry {
+    display: inline-block;
+    font-family: cera-bold;
+    font-size: 20px;
+    width: 300px;
+}
+
+.getArrested {
+    display: inline-block;
+    font-family: cera-bold;
+    font-size:20px;
+    padding-top: 50px;
+    padding-bottom: 20px;
```

Comment:
Padding should be set in `em` units so that the page is responsive. Check out how Dennise did it here: https://github.com/seattleagainstslavery/deterrence-generator/blob/afabb454fbb423b21d0882a0840f560e07fb05b5/src/pages/alternatives.module.scss#L26

Reply:
Changed from px to em units